resources/views/checkout.blade.php

```diff
@@ -0,0 +1,27 @@
+<button
+    id="checkout-{{ $sessionId }}"
+    role="link"
+    style="background-color:#6772E5;color:#FFF;padding:8px 12px;border:0;border-radius:4px;font-size:1em"
+>
+    {{ $label }}
+</button>
+
+<div id="error-message"></div>
+
+<script>
+    const checkoutButton = document.getElementById('checkout-{{ $sessionId }}');
```

Comment:
One thing we've run into is folks putting multiple buttons on the same page. I wonder if for this bit if you want to wrap this in an iife? That way the scope of checkoutButton is isolated to each instance of the checkout view on the page? 

```html
<script>
  (() => {
    const checkoutButton = doc...
   ///
  })()
```